libraries/chain/controller.cpp

```diff
@@ -2612,21 +2612,21 @@ struct controller_impl {
          trx.set_reference_block( self.head_block_id() );
       }
 
-      if (auto dm_logger = get_deep_mind_logger()) {
+      if (auto dm_logger = get_deep_mind_logger(false)) {
```

Comment:
We do not want to log this for dry-run transactions.

Reply:
This is inside `get_on_block_transaction`, which cannot be a transient transaction.